Add flakes tests

vhaldemar opened the pull request.

imakunin reviewed (approved):
I've no idea what this PR do, but all tests a green, so `ship it` :)

vhaldemar commented:
> I've no idea what this PR do, but all tests a green, so `ship it` :)

I added tests for finding unused imports, vars etc.
So you can see how much of imports I deleted.
